tdp_vars_defaults/tdp-cluster/tdp-cluster.yml

```diff
@@ -2,4 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ---
+# Directory where TDP component releases are uploaded for each host 
+upload_tmp_dir: /tmp
```

Comment:
I don't think we should redefine this variable here when it is already defined in `tdp-collection`. If you change the default variable in `tdp-collection` you must not forget to change it here as well since anyway this variable will have priority when merging `tdp_vars`. We can discuss it at our next meeting.